src/js/utils/utils.js

```diff
@@ -18,10 +18,14 @@ function beginsWith(needle, haystack){
     return (haystack.substr(0, needle.length) == needle);
 }
 
+function endsWith(str, suffix) {
+    return str.indexOf(suffix, str.length - suffix.length) !== -1;
```

Comment:
Why not

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith